FABulous/fabric_generator/code_generation_VHDL.py

```diff
@@ -108,20 +108,38 @@ def addLogicStart(self, indentLevel=0):
     def addLogicEnd(self, indentLevel=0):
         self._add("\n" f"end" "\n", indentLevel)
 
-    def addAssignScalar(self, left, right, delay=0, indentLevel=0):
+    def addRegister(self, reg, regIn, clk="CLK", inverted=False, indentLevel=0):
```

Comment:
For large text templates, I think putting the text at the top and then using `.format` should make things look cleaner. We can do this sort of "cosmetic" update later